docs/topics/intellij-idea-plugin-run-configuration.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * docs/intellij-idea-plugin-guide/master.adoc
+
+[id="intellij-idea-plugin-run-configuration_{context}"]
+= Configuring a run configuration
+
+You can configure multiple run configurations to run against each project you import to IntelliJ IDEA.
+
+.Procedure
+
+. In *Projects* view, click the project you want to analyze.
+. On the left side of the screen, click the *{ProductName}* tab.
++
+If this is your first configuration, the run configuration panel is displayed on the right.
+
+. If this is not your first configuration, right-click on one of the configurations in the list and select *New configuration*.
++
+The run configuration panel is displayed on the right.
+
+. Complete the following configuration fields:
+
+** *mta-cli*: Click *Add*, select the filepath, and click *Save*.
+** *Input*: Click *Add* and do one of the following:
```

Comment:
I don't think you need to give two options here. This is pretty elementary. I think the bottom line here is that you need to add an input file or directory. It should be fairly obvious to anyone savvy enough to be reading this document that you can either enter the path manually or using **Open File Explorer**.

```suggestion
** *Input*: Click *Add* and enter the input file or directory.
```

Reply:
Good point. Done.